exchanges/coinbasepro/coinbasepro_websocket.go

```diff
@@ -265,23 +228,6 @@ func (c *CoinbasePro) wsHandleData(ctx context.Context, respRaw []byte) error {
 	return nil
 }
 
-func statusToStandardStatus(stat string) (order.Status, error) {
```

Comment:
suggestion: would caution against this removal and across exchanges that have it, as its a faster lookup. can have these in Unmarshjson methods on exchange defined types. But I understand the consolidation so optional.